joern-cli/frontends/pysrc2cpg/src/main/scala/io/joern/pysrc2cpg/PythonAstVisitorHelpers.scala

```diff
@@ -288,7 +289,13 @@ trait PythonAstVisitorHelpers { this: PythonAstVisitor =>
         .map { case (keyword: String, argNode) => keyword + " = " + codeOf(argNode) }
         .mkString(", ") +
       ")"
-    val callNode = nodeBuilder.callNode(code, name, DispatchTypes.DYNAMIC_DISPATCH, lineAndColumn)
+
+    val passedMethodFullName = methodFullName match {
```

Comment:
I think rather stick to `Optional` types instead of leaving blank strings
```python
    val passedMethodFullName = methodFullName match {
       case Some(fullName) => Option(fullName + name)
       case None           => None
     }
 ```